### DISCUSSION TITLE
Request: Support Darwinia2 with Gnosis Safe.

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#8] [Integration]
https://safe.global/

Now users on Darwinia2 can only use legacy solution from gonosis multisig, it is good, but better to have safe.

**Comment by hackfisher** [comment]
AFAK, @fewensa and @furoxr are working on this.

**Comment by 1022ranji** [comment]
Done. https://multisig.darwiniacommunitydao.xyz/welcome
